src/server/models/products.js

```diff
@@ -0,0 +1,14 @@
+var mongoose = require('mongoose');
```

Comment:
Not sure about this either but as long as it works, maybe ask @Iwuh?

Reply:
Yea i was not sure about this also

Reply:
This looks fine, you have to require mongoose in each model file to get access to mongoose.Schema (mongoose is the package that lets you access the MongoDB database)